scripts/distribution.sh

```diff
@@ -33,9 +33,17 @@ MR_INSTALL_RES_DIR="/usr/local/share/MeshLib"
 # Install the generated bindings, if needed.
 if [ ! -f "distr/meshlib-dev$MR_INSTALL_LIB_DIR/meshlib/mrmeshpy.so" ] && [ -f "build/Release/bin/meshlib/mrmeshpy.so" ]; then
   echo "Installing the generated bindings..."
-  install -Dt "distr/meshlib-dev$MR_INSTALL_LIB_DIR/meshlib" build/Release/bin/meshlib/{mrmeshpy.so,mrmeshnumpy.so,mrcudapy.so,__init__.py}
-  install -Dt "distr/meshlib-dev$MR_INSTALL_LIB_DIR"         build/Release/bin/meshlib/{mrmeshpy.so,mrmeshnumpy.so,mrcudapy.so,__init__.py}
-  patchelf --set-rpath '' "distr/meshlib-dev$MR_INSTALL_LIB_DIR/"{,meshlib/}{mrmeshpy.so,mrcudapy.so}
+  if [ -f "build/Release/bin/meshlib/mrcudapy.so" ]; then
```

Comment:
To my opinion it'd be better to process "mrcudapy.so" and other libraries separately.
```bash
install ... {mrmeshpy.so,mrmeshnumpy.so,__init__.py}
if [ -f "mrcudapy.so" ]; then
    install ... mrcudapy.so
fi
```